### PR TITLE
feat: build static datasources

### PR DESCRIPTION
**Pull request by Eomm**
Relates to #46 

TODO:

- [x] generate `plugins.json` at build time
- [x] generate `benchmarks.json` at build time - requires #43 
- [x] generate `team.json` at build time
- [x] generate `organizations.json` at build time
- [x] generate `acknowledgements.json` at build time

### CODE REVIEW

**Comment by Eomm** [comment]
cc @HassanBahati this PR generates these files that are useful for your `index` PR #38 

- static/generated/organisations.json
- static/generated/team.json
- static/generated/acknowledgements.json

**Comment by HassanBahati** [comment]
> cc @HassanBahati this PR generates these files that are useful for your `index` PR #38
> 
>     * static/generated/organisations.json
> 
>     * static/generated/team.json
> 
>     * static/generated/acknowledgements.json

this is noted. let me pull the changes

File: static/data/organisations.yml [hunk 1]
```diff
@@ -0,0 +1,251 @@
+- name: '2hire'
```
Comment (HassanBahati) [inline static/data/organisations.yml:1]: @Eomm am just curious...what could be the upside of having this data in `.yml` versus `.json`

Comment (HassanBahati) [reply static/data/organisations.yml:1]: i've seen a script then generates the `.json` files....what could could be the intent in first having `yml` then run a script to generate `json`

Comment (Eomm) [reply static/data/organisations.yml:1]: In this way we don't need to update pages/documentation across the organization - it is just another piece of the old website legacy that we could remove after the migration

**Pull request merged**